loomi/reactplate
# feature/linters Add linters config

**What i did**

+ Added and configured `ESLint`, `Prettier` and `stylelint` to check style code and sintax.
+ Added and configured `husky` to run `ESLint`, `Prettier` and `stylelint` on each new commit.
+ Added and configured `lint-staged` to run `husky` only on files that will be commited.

**How to test**

1. Change `''` to `""`, add `console.log()`, etc
2. Save the file and get ready to commit
3. `Husky` will run and check the changed files

**Documentation**
+ _Step by step_ setup: https://thomlom.dev/setup-eslint-prettier-react/
+ `Prettier` options: https://prettier.io/docs/en/options.html
+ `ESLint` rules: https://eslint.org/docs/rules/

+ `stylelint` step by step: https://stylelint.io/user-guide/cli
+ `stylelint`rules: https://stylelint.io/user-guide/rules
